var/spack/repos/builtin/packages/chapel/package.py

```diff
@@ -3,24 +3,813 @@
+        multi=False,
+    )
+
+    variant(
+        "host_platform",
+        description="Host platform",
+        default="unset",
+        values=platform_opts,
+        multi=False,
+    )
+
+    variant(
+        "hwloc",
+        description="Build with hwloc support",
+        default="bundled",
+        values=(
+            "bundled",
+            "none",
+            # CHPL_HWLOC=system existed back to at least 2017,
+            # but it was buggy and unsupported until version 2.1
+            conditional("spack", when="@2.1:"),
+        ),
+        multi=False,
+    )
+
+    variant(
+        "launcher",
+        values=launcher_names,
+        default="unset",
+        description="Launcher to use for running Chapel programs",
+        multi=False,
+    )
+
+    variant(
+        "lib_pic",
+        values=("none", "pic"),
+        default="none",
+        description="Build position-independent code suitable for shared libraries",
+    )
+
+    variant(
+        "libfabric",
+        default="unset",
+        description="When building with ofi support, specify libfabric option",
+        values=("bundled", "spack", "unset"),
+        multi=False,
+    )
+
+    variant(
+        "llvm",
+        default="spack",
+        description="LLVM backend type. The 'spack' value can use an external "
+        "source of LLVM or let spack build a version if no LLVM installs were "
+        "previously detected by 'spack external find'",
+        values=("bundled", "none", "spack"),
+    )
+
+    variant(
+        "re2",
+        description="Build with re2 support",
+        default="bundled",
+        values=("bundled", "none"),
+        multi=False,
+    )
+
+    variant(
+        "target_arch",
+        description="Target architecture for cross compilation",
+        default="unset",
+        values=("x86_64", "aarch64", "arm64", "unset"),
+        multi=False,
+    )
+
+    variant(
+        "target_cpu",
+        values=cpu_options,
+        description="Indicate that the target executable should be specialized "
+        "to the given architecture when using --specialize (and --fast).",
+        default="unset",
+        multi=False,
+    )
+
+    variant(
+        "target_platform",
+        description="Target platform for cross compilation",
+        default="unset",
+        values=platform_opts,
+        multi=False,
+    )
+
+    variant(
+        "tasks",
+        description="Select tasking layer for intra-locale parallelism",
+        default="qthreads",
+        values=("fifo", "qthreads"),
+        multi=False,
+    )
+
+    variant(
+        "timers",
+        description="Select timers implementation",
+        default="unset",
+        values=("generic", "unset"),
+        multi=False,
+    )
+
+    variant(
+        "unwind",
+        description="Build with unwind library for stack tracing",
+        default="none",
+        values=("bundled", "none", "spack"),
```

Comment:
(minor) I don't see the logic handling a distinction between `none` and `bundled` for this particular variant.